<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Medusa</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MedusaRsrc.qrc">
    <normaloff>:/images/medusa_logo.png</normaloff>:/images/medusa_logo.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuLabels">
     <property name="title">
      <string>Labels</string>
     </property>
     <addaction name="actionData"/>
     <addaction name="actionCode"/>
     <addaction name="actionString"/>
     <addaction name="actionImported"/>
     <addaction name="actionExported"/>
    </widget>
    <addaction name="actionDisassembly"/>
    <addaction name="separator"/>
    <addaction name="actionJump_History"/>
    <addaction name="actionLog"/>
    <addaction name="separator"/>
    <addaction name="menuLabels"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionGoto"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionDisassembly"/>
  </widget>
  <widget class="QDockWidget" name="logDock">
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPlainTextEdit" name="logEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximumBlockCount">
        <number>500</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dataLabelDock">
   <property name="windowTitle">
    <string>Data Labels</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QListWidget" name="dataList">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="codeLabelDock">
   <property name="windowTitle">
    <string>Code Labels</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QListWidget" name="codeList">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="stringLabelDock">
   <property name="windowTitle">
    <string>String Labels</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QListWidget" name="stringList">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="importedLabelDock">
   <property name="windowTitle">
    <string>Imported Labels</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QListWidget" name="importedList">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="exportedLabelDock">
   <property name="windowTitle">
    <string>Exported Labels</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QListWidget" name="exportedList">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="MedusaRsrc.qrc">
     <normaloff>:/images/OpenFile.png</normaloff>:/images/OpenFile.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="MedusaRsrc.qrc">
     <normaloff>:/images/CoseFile.png</normaloff>:/images/CoseFile.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="MedusaRsrc.qrc">
     <normaloff>:/images/About.png</normaloff>:/images/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDisassembly">
   <property name="icon">
    <iconset resource="MedusaRsrc.qrc">
     <normaloff>:/images/Disassembly.png</normaloff>:/images/Disassembly.png</iconset>
   </property>
   <property name="text">
    <string>Disassembly</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionHexDump">
   <property name="text">
    <string>Hex Dump</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="MedusaRsrc.qrc">
     <normaloff>:/images/goto.png</normaloff>:/images/goto.png</iconset>
   </property>
   <property name="text">
    <string>Goto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="MedusaRsrc.qrc">
     <normaloff>:/images/Gear.png</normaloff>:/images/Gear.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionJump_History">
   <property name="text">
    <string>Jump History</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionData">
   <property name="text">
    <string>Data</string>
   </property>
  </action>
  <action name="actionCode">
   <property name="text">
    <string>Code</string>
   </property>
  </action>
  <action name="actionString">
   <property name="text">
    <string>String</string>
   </property>
  </action>
  <action name="actionImported">
   <property name="text">
    <string>Imported</string>
   </property>
  </action>
  <action name="actionExported">
   <property name="text">
    <string>Exported</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MedusaRsrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLog</sender>
   <signal>triggered(bool)</signal>
   <receiver>logDock</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>646</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionData</sender>
   <signal>triggered(bool)</signal>
   <receiver>dataLabelDock</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCode</sender>
   <signal>triggered(bool)</signal>
   <receiver>codeLabelDock</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionString</sender>
   <signal>triggered(bool)</signal>
   <receiver>stringLabelDock</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExported</sender>
   <signal>triggered(bool)</signal>
   <receiver>exportedLabelDock</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>691</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImported</sender>
   <signal>triggered(bool)</signal>
   <receiver>importedLabelDock</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
